doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -458,7 +458,8 @@ Enabling and disabling notations
 
    Enables or disables notations previously defined with
    :cmd:`Notation` or :cmd:`Notation (abbreviation)`.
-   It has no effect on notations reserved with :cmd:`Reserved Notation`.
+   Disabling a notation doesn't remove parsing rules or tokens defined by the notation.
+   The command has no effect on notations reserved with :cmd:`Reserved Notation`.
```

Comment:
I would even say:
```suggestion
   Thus, the command has no effect on notations reserved with :cmd:`Reserved Notation`.
```

Reply:
We already had that discussion with Jim (c.f., https://github.com/coq/coq/pull/16812#discussion_r1023851630 just above but folded), so I'd say let's keep it as is.

Reply:
OK, sorry I didn't see.

Reply:
No worries, that was actually hidden.